testfiles/PlantLoopHeatPump_EIR_AirSource_and_AWHP_cooling_only.idf

```diff
@@ -0,0 +1,497 @@
+! PlantLoopHeatPump_EIR_AirSource.idf
+!
+! Basic file description:
+! The EIR formulated air-to-water heat pump is demonstrated using a simple collection of plant loops.  Building loads
+! are represented using Plant Load Profile objects
+! Run:      1 design day.
+! Building: None.
+! System:   None.
+! Plant:    PLANT LOAD PROFILE with District Heating and Cooling as necessary.
```

Comment:
The three testfiles added have exactly the same description. I feel like there should be be some difference here.

And the mention of the special shenanigans like the EMS program would probably be good for the heating/cooling only ones.

Reply:
Indeed. I forgot to update the description. I will modify them.

Reply:
Thanks for catching this and for all your review! @jmarrec